app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventSelector/EventListAdapter.java

```diff
@@ -0,0 +1,67 @@
+package ch.epfl.sweng.eventmanager.ui.eventSelector;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import ch.epfl.sweng.eventmanager.R;
+import ch.epfl.sweng.eventmanager.repository.data.Event;
+import ch.epfl.sweng.eventmanager.ui.eventShowcase.EventActivity;
+
+import java.util.List;
+
+import static android.content.ContentValues.TAG;
+
+public class EventListAdapter extends RecyclerView.Adapter<EventListAdapter.ViewHolder> {
+    private List<Event> mEvents;
+
+    // Provide a reference to the views for each data item
+    public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+        // each data item is just a string in this case
+        public TextView eventNameTextView;
+        public ViewHolder(View itemView) {
+            super(itemView);
+            itemView.setOnClickListener(this);
+            eventNameTextView = (TextView) itemView.findViewById(R.id.event_name);
+        }
+
+        @Override
+        public void onClick(View view) {
+            // TODO: display animation on click?
+            Context context = itemView.getContext();
+            Intent intent = new Intent(context, EventActivity.class);
+            Event selectedEvent = mEvents.get(getAdapterPosition());
+            intent.putExtra(EventPickingActivity.SELECTED_EVENT_ID, selectedEvent.getId());
```

Comment:
Don't we want to pass directly the Event for now?

Reply:
We currently get the event back from the ID in the `ui.eventShowcase.EventActivity` activity: I think it's enough for the current sprint, but we indeed have to think harder for the next sprint.